tests/cli/helpers.go

```diff
@@ -18,6 +18,13 @@ import (
 )
 
 var (
+	TotalCoins = sdk.NewCoins(
```

Comment:
We should migrate these consts into functions (tests run in parallel and we want to avoid any sort of race condition)

Reply:
Could be done in another PR though

Reply:
Sure, I'll add functions to return these values in next PR